leakybucket.go

```diff
@@ -541,8 +568,9 @@ func NewLeakyBucketMemcached(cli *memcache.Client, key string, ttl time.Duration
 func (t *LeakyBucketMemcached) State(ctx context.Context) (LeakyBucketState, error) {
 	var item *memcache.Item
 	var err error
-	state := LeakyBucketState{}
+	var stateWithTTL leakyBucketStateWithExpirationTime
 	done := make(chan struct{}, 1)
+	t.casId = 0
```

Comment:
Please add a comment explaining why setting casId to zero is needed here.

Btw, is it really needed? If the read operation succeeds then `casId` will be set to `item.CasID` below, if the read operation fails then the last successfully read `casId` value will be kept (as it probably should). Am i missing something?